VideoAPI/VideoApi.DAL/Commands/RemoveRoomParticipantCommand.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Linq;
+using VideoApi.DAL.Commands.Core;
+using VideoApi.DAL.Exceptions;
+using Task = System.Threading.Tasks.Task;
+
+namespace VideoApi.DAL.Commands
+{
+    public class RemoveRoomParticipantCommand : ICommand
+    {
+        public long RoomId { get; }
+        public Guid ParticipantId { get; }
+
+        public RemoveRoomParticipantCommand(Guid participantId, long roomId)
+        {
+            ParticipantId = participantId;
+            RoomId = roomId;
+        }
+    }
+
+    public class RemoveRoomParticipantCommandHandler : ICommandHandler<RemoveRoomParticipantCommand>
```

Comment:
Do you need `AssignLeaveTimeForRoomParticipantCommand` if you already have this command?

Reply:
Correct, I don't need. Done

Reply:
done, assign leave time is removed